packages/core/src/lib/select/select-input.svelte

```diff
@@ -69,6 +70,7 @@ $: errorClasses =
     type="text"
     class={cx(
       'h-7.5 w-full grow appearance-none border py-1.5 pl-2 pr-1 text-xs leading-tight outline-none',
+      { 'pl-8': Boolean(icon) },
```

Comment:
Used to give space for the icon. Would welcome (with open arms) a better way to do this

Reply:
Since we want the native select to visually include the icon, we will need absolute positioning and this padding.

Do you mind leaving a comment to describe why this padding is necessary?

Reply:
27878cc